src/o365/spfx/commands/project/project-upgrade/model/ConfigJson.ts

```diff
@@ -6,10 +6,19 @@ export interface ConfigJson {
   entries?: Entry[];
   localizedResources?: Hash;
   version?: string;
+  externals?:ExternalConfiguration;
 }
 
 export interface Entry {
   entry: string;
   manifest: string;
   outputPath: string;
+}
+export interface ExternalConfiguration {
+  [key: string]: External;
+}
+export interface External {
+  path: string;
+  globalName?: string;
```

Comment:
According to config.json schema, globalName is required.

Reply:
fixed